types/accounting/index.d.ts

```diff
@@ -1,6 +1,7 @@
-// Type definitions for accounting.js 0.3
+// Type definitions for accounting.js 0.4
 // Project: http://josscrowcroft.github.io/accounting.js/
```

Comment:
The site is dead now, probably we also need to update the link to the http://openexchangerates.github.io/accounting.js/